### ISSUE TITLE
Möglichkeit eines reinen REST API-Nutzers, der sich nicht per GUI einloggen darf

### ISSUE THREAD

**Issue by GuidoBo** [jlawyerorg/j-lawyer-org#1321]
Über die Nutzer-Funktion haben wir einen Benutzer (swagger) angelegt, dessen Rechte beschränkt sind auf Datenimport, Adressen erstellen, Akten erstellen. Mit diesem Nutzer klappt der API-Test über die swagger-ui nicht, solange in den Berechtigungen "Login" fehlt.

Wenn es dann aber um den späteren Import von Fällen geht: kann dann das Recht "Login" wieder gelöscht werden (wäre toll).

**Comment by GuidoBo** [comment]
sorry; das war wohl uneindeutig: kann der Nutzer **ohne Login noch Daten importieren**?

**Comment by j-dimension** [comment]
Nein, ohne Login-Recht darf niemand irgendetwas. Was wollt Ihr erreichen - einen Nutzer haben, der ausschließlich für die API-Aufrufe genutzt werden kann?

**Comment by GuidoBo** [comment]
Genau das war die Idee.

**Comment by j-dimension** [comment]
@GuidoBo okay, ich habe das Ticket umformuliert und öffne es wieder.
